db/migrate/20210603122414_rename_allocation_to_allocation_history.rb

```diff
@@ -0,0 +1,16 @@
+class RenameAllocationToAllocationHistory < ActiveRecord::Migration[6.0]
+  def change
+    rename_table :allocations, :allocation_history
```

Comment:
what was wrong with the default allocation_histories table name?

Reply:
No strong reason: I didn't think 'histories' made sense in this context, so thought an "allocation_history" table would be more readable.

Although I've just seen that we have an "email_histories" table, so maybe it's better to be consistent and stick to the Rails convention.

I'm happy to change this back if you think it's worth doing?